cmd/policy.go

```diff
@@ -83,12 +84,29 @@ func (sys *PolicySys) Init(objAPI ObjectLayer) error {
 	for _, bucket := range buckets {
 		config, err := GetPolicyConfig(objAPI, bucket.Name)
 		if err != nil {
-			if _, ok := err.(BucketPolicyNotFound); !ok {
+			if _, ok := err.(BucketPolicyNotFound); ok {
```

Comment:
You better keep existing check.  This change is not required.

Reply:
why I have to handle the BucketPolicyNotFound again in subsequent line. This is cleaner to communicate

Reply:
Why do you need to check BucketPolicyNotFound again in `else` block?  cleaner to communicate is a perception.

Reply:
I reverted back to keep the minimal change, `if else` blocks are a bit weird it is harder to read `not yes` v/s `yes` - avoids unnecessary code indentation and communicates why we are trying to skip BucketPolicyNotFound.  But in lieu of moving forward, I will keep it as is.